pyanaconda/installation.py

```diff
@@ -127,13 +134,6 @@ def _prepare_configuration(payload, ksdata):
                                    network.write_configuration, (overwrite, )))
         configuration_queue.append(network_config)
 
-    # add installation tasks for the Users DBus module
-    user_config = TaskQueue("User creation", N_("Creating users"))
-    users_proxy = USERS.get_proxy()
-    users_dbus_tasks = users_proxy.InstallWithTasks()
-    os_config.append_dbus_tasks(USERS, users_dbus_tasks)
```

Comment:
It is a little difficult to spot the problem. The task queue was messed up in the commit 8e5fac37263e89a9b08d677a06cf9695e5eaf9c7. However, I don't understand why do you also change the order of the network configuration and user configuration queues.

Reply:
Ah, sure, to keep the order from before the https://github.com/rhinstaller/anaconda/commit/8e5fac37263e89a9b08d677a06cf9695e5eaf9c7 I should not change it. I just didn't want to change the order now to be on the safer side but it doesn't seem to matter anyway so I'll update the PR.

Reply:
Done